tools/android/packaging/Makefile.in

```diff
@@ -102,7 +102,7 @@ python: | xbmc/assets
 	cd xbmc/assets/python2.7/lib/python2.7/; rm -rf test config lib-dynload
 
 res:
-	mkdir -p xbmc/res xbmc/res/raw xbmc/res/values images
+	mkdir -p xbmc/gen/org/xbmc/kodi xbmc/res xbmc/res/raw xbmc/res/values images
```

Comment:
Those are temp files. there is no added value to use APP_NAME, really.
To be paralleled with the src/org/xbmc/kodi/ folder, which is not parametrized (and does not need to)